src/example/zalog_example_prog_rep_logger.prog.abap

```diff
@@ -3,7 +3,12 @@ REPORT zalog_example_prog_rep_logger.
 INITIALIZATION.
   DATA(go_logger) = NEW zcl_alog_progress_rep_logger( ).
 
+* EP Set Total Entries
+  go_logger->set_TOTAL_ENTRIES( 10 ).
```

Comment:
What does `EP` stand for? The comment also seems really unnecessary.

Also please pretty print anything non-statement to lower-case.

Reply:
fixed. just for your info, i always sign my code with "EP" it's my initial letter :-)